script/create-dist

```diff
@@ -59,6 +59,12 @@ BINARIES = {
     'libdevtools_discovery.a',
     'libdevtools_http_handler.a',
     'libhttp_server.a',
+    'libdesktop_capture.a',
```

Comment:
The `libdesktop_capture_differ_sse2` are needed in `webrtc` on Windows and Linux.